### ISSUE TITLE
Crashes when tapping on update Google Play Services notification

### ISSUE THREAD

**Issue by rugk** [corona-warn-app/cwa-app-android#506]
## Describe the bug

What happens:
* You first get the error as in https://github.com/corona-warn-app/cwa-app-android/issues/472
* At the next restart, you get up to the "Activate collection" screen.
* You see a notification of the app: "Please update your Google Play Services."
* You tap on that.
* -> This app crashes (You may see the Android screen "Corona-App[…] has stopped."
* Also, no intend seems to be actually send or so? At least no app store did open in my case… :roll_eyes: 

## Expected behaviour

Not crash.

## Steps to reproduce the issue

1. Have not updated Google Play Services.
2. Open the app.

## Technical details

- Host Machine OS (Windows/Linux/Mac): ??? Android… why host?

Android 9

## Possible Fix

You may exit the app, but not crash…
Do it a little more gracefully…

## Additional context

Google Play is disabled on the device. Using Aurora Store for updating.

---
Internal Tracking ID: EXPOSUREAPP-1979

**Comment by HrGaertner** [comment]
I have something similar i get this message the message opens the google play store but my google play services are already up to date (android 8.1.0)

**Comment by genodeftest** [comment]
Same issue here. Relevant parts from `adb logcat`:
```
06-20 12:39:00.482  2629  5804 I ActivityManager: START u0 {act=android.intent.action.VIEW dat=market://details?id=com.google.android.gms&pcampaignid=gcore_12451000--de.rki.coronawarnapp-27 flg=0x80000 pkg=com.android.vending} from uid 10309
06-20 12:39:00.484 19094 19094 D AndroidRuntime: Shutting down VM
--------- beginning of crash
06-20 12:39:00.491 19094 19094 E AndroidRuntime: FATAL EXCEPTION: main
06-20 12:39:00.491 19094 19094 E AndroidRuntime: Process: de.rki.coronawarnapp, PID: 19094
06-20 12:39:00.491 19094 19094 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{de.rki.coronawarnapp/com.google.android.gms.common.api.GoogleApiActivity}: android.content.ActivityNotFoundException: No Activity found to handle null
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2928)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3063)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1823)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:107)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:198)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6729)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: Caused by: android.content.ActivityNotFoundException: No Activity found to handle null
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:2014)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.app.Activity.startIntentSenderForResultInner(Activity.java:4845)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.app.Activity.startIntentSenderForResult(Activity.java:4812)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.app.Activity.startIntentSenderForResult(Activity.java:4779)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at com.google.android.gms.common.api.GoogleApiActivity.onCreate(com.google.android.gms:play-services-base@@17.3.0:12)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:7136)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:7127)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1272)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2908)
06-20 12:39:00.491 19094 19094 E AndroidRuntime: 	... 11 more
06-20 12:39:00.495  2629  5804 W ActivityManager:   Force finishing activity de.rki.coronawarnapp/com.google.android.gms.common.api.GoogleApiActivity
06-20 12:39:00.496  2629  5804 D ActivityTrigger: ActivityTrigger activityPauseTrigger 
06-20 12:39:00.499  2629  2727 D StatusBarManagerService: manageDisableList userId=0 what=0x0 pkg=Window{cc57ce2 u0 StatusBar}
06-20 12:39:00.499  2629  2727 D StatusBarManagerService: disableLocked net1 = 0; net2 = 0 pkg=Window{cc57ce2 u0 StatusBar}
06-20 12:39:00.499  3033  3033 D StatusBar: hiding the MENU button
06-20 12:39:00.501  2629  2727 I ActivityManager: Showing crash dialog for package de.rki.coronawarnapp u0
```

I can provide more detailed logs via a non-public channel (e.g. e-mail) on request.

**Comment by jakobmoellersap** [comment]
The activity that is used for navigation might not be supported by Aurora Store. Can someone confirm this?

**Comment by genodeftest** [comment]
@jakobmoellersap wrote:
> The activity that is used for navigation might not be supported by Aurora Store. Can someone confirm this?

I have installed corona warn app through Aurora Store (never used Google Play Store at all) too. How can this be related to the app store though? I don't see issues with any other notification of any other app.

Since I have updated Google Play Services and CWA (with #472 fixed), I don't see the crash any more. I extracted a more detailed log to my PC's disk though.
